RocketChat/Rocket.Chat
# Added placeholder to password field in tr

Proposed changes
I noticed that the password field in the login form didn't have a placeholder, which could make the interface a bit confusing for first-time users. This Pull Request adds a simple placeholder text ("Enter your password") to make it clear what the input field is for. It’s a small change, but it enhances the overall user experience.

Issue(s)
This change doesn't directly address any existing issues but aligns with improving the user experience.

Steps to test or reproduce
Open the login form in the application.
Look at the password input field.
Verify that the placeholder text "password" is visible in the field before the user starts typing.
Further comments
Adding this placeholder makes the login form a bit more user-friendly, especially for new users. It’s a subtle improvement, but sometimes the little details make a big difference. Let me know if there's anything else I should tweak or improve!
